### PR TITLE
Contributed my part of the story

### PR DESCRIPTION
**Pull request by NishchithAnanthaRamu**

### CODE REVIEW

**Comment by cmoore24361** [comment]
Looks pretty good! I am not sure if it matters, but Arushi and I have the madlibs_list.txt entries as something like "large" instead of "adjective : Large". 
Also, to get full points, make sure to review and comment on Nikiles's pull request. After y'all have done that for each other, you can merge your own commits.

**Review by nikiles3 — APPROVED** [review]
Looks good!

**Comment by nikiles3** [comment]
Nishchith your merge request was approved!

**Comment by cmoore24361** [comment]
I just noticed, placeholder [16] never got updated in the story, just the madlibs_list.txt

**Comment by nikiles3** [comment]
> I just noticed, placeholder [16] never got updated in the story, just the madlibs_list.txt

Can you check now. It was just updated.

File: madlibs_list.txt
Comment (nikiles3) [inline madlibs_list.txt:1]: Nishchith, your changes have been reviewed and merged.

**Comment by arushi3103** [comment]
Hey Y'all, just to confirm, none of us have any pending approvals, right? I am going by the announcement Professor shared a while back.
I cannot see any approval pending for me, but if there is any, please let me know.

**Comment by cmoore24361** [comment]
Nothing pending approval, but I think I fall into who he was talking about. I don't think I ever "reviewed" a pull request, and we can only review active pull requests. I think you did though, so you are good.

**Pull request merged**